portal-api/src/v1/users/validation/rules/select-at-least-one-bank.js

```diff
@@ -0,0 +1,37 @@
+const { ADMIN } = require('../../../roles/roles');
+
+const CONSTANT = require('../../../../constants');
+/**
+ * Validates that if the bank is present it is not empty
+ * @param {Object} user the existing user
+ * @param {Object} change the changes to make
+ * @returns {Array} either an empty array or an array containing an error object
+ */
+const selectAtLeastOneBank = (user, change) => {
+  if (!change?.bank || change.bank.length === 0) {
```

Comment:
This still needs to be completed.

Reply:
When I changed to `if (!change?.bank?.length === 0`) I see one of the test failing. Not sure if chaining is done correctly.
![image](https://github.com/UK-Export-Finance/dtfs2/assets/113366256/fb74f68a-fe1e-42c4-a90a-293e32f42daa)